deployments/ansible/playbook.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Call this something like `example-playbook.yml` so that it is clear this is for development.  People that are actually using this role won't need a playbook from us.

Reply:
updated